tool/tctl/common/tctl.go

```diff
@@ -754,39 +758,69 @@ func (a *AuthCommand) GenerateAndSignKeys(client *auth.TunClient) error {
 		return trace.Wrap(err)
 	}
 
-	certPath := a.genUser + "-cert.pub"
-	keyPath := a.genUser
-	pubPath := a.genUser + ".pub"
-
-	// --out flag
-	if a.outDir != "" {
-		if !utils.IsDir(a.outDir) {
-			if err = os.MkdirAll(a.outDir, 0770); err != nil {
+	switch a.outputFormat {
+	//
+	// dump user identity into a single file:
+	//
+	case IdentityFormatFile:
+		if a.output == "" {
+			a.output = fmt.Sprintf("%s.pem", a.genUser)
+		}
+		var (
+			output  io.Writer
```

Comment:
This isn't really consistent with how `tctl` has worked up to this point, `tctl auth export` and `tctl saml export` output to `stdout` and we redirect to a file.